src/DomainServer.ts

```diff
@@ -343,41 +343,32 @@ class DomainServer {
         if (node.getType() === NodeType.EntityServer) {
             console.warn("DomainServer: EntityServer support implemented!");
 
-            // WEBRTC TODO: Address further code - for AssetServer node.
+            // WEBRTC TODO: Address further code - for EntityServer node.
 
         }
     };
 
     // Slot.
     #nodeActivated = (node: Node): void => {
-        // C++  void Application:: nodeActivated(Node* node)
+        // C++  void Application::nodeActivated(Node* node)
         const nodeType = node.getType();
 
+        // AudioMixer node is handled in AudioMixer.ts.
+
         if (nodeType === NodeType.AssetServer) {
             console.warn("DomainServer: AssetServer support not implemented!");
 
             // WEBRTC TODO: Address further code - for AssetServer node.
 
-        }
-
-        if (nodeType === NodeType.EntityServer) {
+        } else if (nodeType === NodeType.EntityServer) {
```

Comment:
This series of 'else if's should probably become a switch-case statement as it will get longer and a switch would be more readable.

Reply:
Per Application::nodeKilled() in the C++, it shouldn't get any longer. In actual, fact it will quite likely get shorter: Application.cpp has 3 linked `if`s but the TypeScript has only 2 because the `AudioMixer` one has been moved to AudioMixer.ts. (The two remaining ones will quite likely move to their audio mixer TypeScript files once implemented.)